platform/ios/src/MGLMapView.h

```diff
@@ -390,6 +390,13 @@ IB_DESIGNABLE
  */
 @property(nonatomic) CGFloat decelerationRate;
 
+/**
+ The gesture recognizer used to detect single taps on the map.
+ 
+ Your application can use this to modify the behavior of your own gesture recognizers.
+ */
+@property (nonatomic, readonly) UITapGestureRecognizer *singleTapGestureRecognizer;
```

Comment:
Is it enough for the application's gesture recognizers to avoid conflicts with this one gesture recognizer, or is it also necessary to avoid conflicts with, say, the double-tap gesture recognizer? Would it make sense for the application's gesture recognizer to require all the map view gesture recognizers (via the `gestureRecognizers` property) to fail? Or would those other gesture recognizers need to defer to the application's single tap gesture recognizer?